src/main/java/fact/features/singlePulse/timeLineExtraction/AddFirstArrayToSecondArray.java

```diff
@@ -0,0 +1,21 @@
+package fact.features.singlePulse.timeLineExtraction;
+
+import java.util.Arrays;
+
+public class AddFirstArrayToSecondArray {
+    
+    public static void at(double[] f1, double[] f2, int at) {
+        if(at > f2.length) return;
+        if(at < 0) at = 0;
+        // endpoint of injection in f2: e2
+        int e2 = at + f1.length;
+        // naive endpoint of sampling in f1
+        int e1 = f1.length;
```

Comment:
This turned out to be a bug. I tracked it down with two more test cases.
Looks like I missed sth. from the python port.